---
title: "Omnibio"
date: 2017-11-18
draft: true
---



Boyle, Li, and Pritchard wrote a paper that's been getting a lot of
buzz on twitter recently.
[An Expanded View of Complex Traits: From Polygenic to Omnigenic](http://dx.doi.org/10.1016/j.cell.2017.05.038)
names the "omnigene" hypothesis. 
It's also a case study in how to twitter promote your paper.

There's been a lot of disappointment in the inability of GWAS to
ascribe all the variation in traits to genetic variation, with a 
lot of missing heritability and effects of very small sizes.
The advance here is accepting that what we measure in GWAS is sorta
how it is, that there's a set of few core genes that "tend to have
biologically interpretable roles in disease" and a numerous periphery
of genes with small effects. In their analysis, it's 62% of all
common SNPs that associate with an effect on height.
Here, they suppose that this periphery
has it's effect through mediating the expression of the core genes.

What are the implications for evolution? They outline that it'll be
damn hard to track the evolution of adaptive allele unless they're
part of that "core". This makes sense that experimental evolution
that looks at the first adaptations is going to pick up the big
core effects, and sets us up nicely to use barcode lineage
tracking (like with 
[Levy and Blundell et al. 2015](http://dx.doi.org/10.1038/nature14279) 
) to look at second steps and the distribution of small effects.

I hope a careful survey of the diversity and order of trajectories
in carefully controlled experimental evolution can shed light on
these problems.
